package.json

```diff
@@ -28,10 +32,16 @@
     "atom": ">=1.0.0 <2.0.0"
   },
   "dependencies": {
-    "atom-languageclient": "0.8.3",
+    "atom-languageclient": "^0.8.3",
     "atom-package-deps": "^4.6.1",
+    "deepmerge": "^2.0.1",
+    "fs-extra": "^5.0.0",
     "ocaml-language-server": "1.0.27"
   },
+  "activationCommands": {
+    "atom-workspace": "ide-reason:generate-config"
+  },
+  "activationHooks": ["language-reasonml:grammar-used", "language-ocaml:grammar-used"],
```

Comment:
`"language-reasonml:grammar-used" -> "language-reason:grammar-used"`

Without `ml`.

Reply:
Just finded in another pr...I'll fix it several hours later.